3rd-party/beaverbuilder.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Beaverbuilder Compatibility.
  */
-class BeaverBuilderCompat {
```

Comment:
What do you think about using something like `Jetpack_BeaverBuilderCompat` instead, just to be sure this does not happen again in the future?

Reply:
Yea im happy with that, i'll update the pr.